doc/topics/tutorials/gitfs.rst

```diff
@@ -555,6 +557,15 @@ single branch.
       - http://foo.com/quux.git:
         - all_saltenvs: anything
 
+If your want to also test working branches of the formula repository, use
+``fallback`` and ensure you branch names don't collapse between repositories:
```

Comment:
I agree, this was unclear. I meant: use non-ambiguous names (i.e don't use `feature/wip1`, but use `feature/the-feature-name`), to avoid matching a branch with the same name in another repo. But sometimes you want exactly this (in this case, use the same branch name).